### PR TITLE
Supporting table mapping

### PR DESCRIPTION
**Pull request by Paultagoras**
This adds a new setting to allow customers to specify a mapping between topic and table names (without a transformer)

### CODE REVIEW

File: src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java [hunk 1]
```diff
@@ -161,11 +161,13 @@ public void doInsert(List<Record> records, ErrorReporter errorReporter) {
         try {
             Record first = records.get(0);
             String topic = first.getTopic();
-            Table table = this.mapping.get(Utils.escapeTopicName(topic));
+//            Table table = this.mapping.get(Utils.escapeTopicName(topic));
```
Comment (mzitnik) [inline src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java:164]: Lets remove it

**Review by mzitnik — CHANGES_REQUESTED** [review]
Please provide also tests

File: src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java [hunk 2]
```diff
@@ -523,7 +525,7 @@ public void doInsertJson(List<Record> records) throws IOException, ExecutionExce
         Record first = records.get(0);
         String topic = first.getTopic();
         LOGGER.info(String.format("Number of records to insert %d to table name %s", batchSize, topic));
-        Table table = this.mapping.get(Utils.escapeTopicName(topic));
+        Table table = this.mapping.get(Utils.getTableName(topic, csc.getTopicToTableMap()));
```
Comment (mshustov) [inline src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java:528]: What about `describeTable`? We need to get metadata from another table  https://github.com/ClickHouse/clickhouse-kafka-connect/blob/374227d099367c0ae8480a4fae6dd8f63d5f90b6/src/main/java/com/clickhouse/kafka/connect/sink/db/helper/ClickHouseHelperClient.java#L135

File: src/main/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkConfig.java [hunk 3]
```diff
@@ -23,6 +23,7 @@ public class ClickHouseSinkConfig {
     public static final String EXACTLY_ONCE = "exactlyOnce";
     public static final String SUPPRESS_TABLE_EXISTENCE_EXCEPTION = "suppressTableExistenceException";
     public static final String CLICKHOUSE_SETTINGS = "clickhouseSettings";
+    public static final String TABLE_MAPPING = "tableMapping";
```
Comment (mshustov) [inline src/main/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkConfig.java:26]: in bigquery connector it's called `topic2TableMap, in snowflake connector `topic2table.map`

**Review by mzitnik — APPROVED** [review]

**Pull request merged**